src/app/base/components/node/NodeTestDetails/NodeTestDetails.test.tsx

```diff
@@ -47,10 +47,12 @@ describe("NodeTestDetails", () => {
       <Provider store={store}>
         <MemoryRouter initialEntries={["/machine/abc123/testing/1/details"]}>
           <CompatRouter>
-            <Route
-              path="/machine/:id/testing/:scriptResultId/details"
-              render={() => <NodeTestDetails getReturnPath={getReturnPath} />}
-            />
+            <Routes>
+              <Route
+                element={<NodeTestDetails getReturnPath={getReturnPath} />}
+                path="/machine/:id/testing/:scriptResultId/details"
```

Comment:
RRv6 `useParams` needs to have the route set up to be able to capture the params so a lot of tests had to be updated to include `<Route>`